helm/cluster/files/etc/containerd/config.toml

```diff
@@ -0,0 +1,50 @@
+version = 2
```

Comment:
This is just copy-pasted from `cluster-aws` (it's literally the same in all providers I think) with only Helm values being adjusted to the `cluster` chart.